src/registry/registry.json

```diff
@@ -1429,20 +1429,23 @@
       "name": "Territory2Model",
       "suffix": "territory2Model",
       "directoryName": "territory2Models",
+      "inFolder": false,
       "folderType": "territory2model"
     },
     "territory2rule": {
       "id": "territory2rule",
       "name": "Territory2Rule",
       "suffix": "territory2Rule",
       "directoryName": "rules",
+      "inFolder": false,
       "folderType": "territory2model"
     },
     "territory2": {
       "id": "territory2",
       "name": "Territory2",
       "suffix": "territory2",
       "directoryName": "territories",
+      "inFolder": false,
```

Comment:
Adding these back since they are returned by a describemetadata call, and that's what we should be using as the source of truth for type definitions.

Reply:
source of untruth